src/Lists.php

```diff
@@ -267,7 +267,7 @@ public function checkListMembership( $listId, array $emails = null, array $phone
             )
         );
 
-        $path = sprintf('%s/%s/%s', self::LIST, $listId, self::MEMBERS );
+        $path = sprintf('%s/%s/%s', self::ENDPOINT_LIST, $listId, self::MEMBERS );
```

Comment:
MEMBERS constant got changed. looks like all the endpoints might need an update.